### ISSUE TITLE
Exporting from preview panel does not use "Preview ratio" setting

### ISSUE THREAD

**Issue by pegep** [gephi/gephi#1567]
Preview ratio is not taken into account when exporting from Preview panel. The whole graph is exported regardless of the "Preview ratio" setting. Export should export only the graph that is visible in the preview.

This was reported in FB group: https://www.facebook.com/groups/gephi/permalink/10154992658447565/

**Comment by eduramiba** [comment]
Preview ratio is actually used to reduce the time needed to render a **preview** of what is actually exported later. It randomly removes data from the graph, why do you need that in an exported file?

**Comment by pegep** [comment]
Yes, I understand that. However for a user it might be confusing if the export is exporting something different than is on the screen. It's not completely randomly removing data either as it seems to remove the lowest degree nodes first. I reported this for Mika from FB because it makes sense to me too that the export would export something that is on the preview panel.

The use case is that the user wants to export data with 10%, 20%, ... 100% to produce a sequence of files. I recommended using filtering options but it's not exactly the same.
